dataprofiler/tests/profilers/test_utils.py

```diff
@@ -44,3 +44,42 @@ def test_uneven_chunk_sample(self):
             num_chunks += 1
         self.assertEqual(num_chunks, 100 // 7 + 1)
         self.assertCountEqual(all_values, list(range(100)))
+        
+    def test_find_diff(self):
+        """
+        Checks to see if the find difference function is operating
+        appropriately.
+        """
+
+        # Ensure lists and sets are handled appropriately
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets([3, 2], [2, 3, 2]))
```

Comment:
This feels like it this could be considered a "change", I don't know what customers said their use cases were, but maybe lists take count into consideration? Idk I am not dead set on this there can be a conversation but I think this should have the difference as `[2]`.

Reply:
I think the things you mentioned can be covered from sample_size and word_counts/category_counts @ChrisWallace2020

Reply:
@ChrisWallace2020 , if you can find me an example in our report, I would change it but otherwise, I think sets and lists should be treated the same.